README.md

```diff
@@ -285,6 +285,8 @@ services:
       KIOSK_PREFETCH: true
       KIOSK_ASSET_WEIGHTING: true
       KIOSK_PORT: 3000
+      KIOSK_IMMICH_USERS_API_KEYS: "{\"user1\":\"****\"}"
```

Comment:
I would remove this as an option. I think it's ok if not everything is configurable via ENV, especially advanced settings. Webhooks, weather and redirects are only configurable via a yaml file.

Reply:
sounds good 👌